### ISSUE TITLE
[Feature] Remove benches/trash can at PillBox

### ISSUE THREAD

**Issue by BigSmashGG** [FrenzyRoleplay/server-issues#642]
The benches are constantly hit and glitch themselves sometimes blocking the door at pillbox.

removing them all together would help with the issues it causes there.

Also removing the trash can by the front door, same problem.

**Comment by nixety** [comment]
Is this still an issue?

**Comment by BigSmashGG** [comment]
yes. would like to have the benches and the trash cans removed from in front of pillbox doors
